src/helpers/actions.ts

```diff
@@ -76,13 +76,13 @@ export function refreshVotesCount(spaces: string[]) {
     `
       INSERT INTO leaderboard (vote_count, user, space)
         (SELECT * FROM (
-          SELECT COUNT(votes.id) AS vote_count, voter, space
+          SELECT COUNT(votes.id) AS vote_count, MAX(created) as last_vote, voter, space
```

Comment:
while running the script i get this error 

```bash
OperationalError: ER_NON_UNIQ_ERROR: Column 'created' in field list is ambiguous
    at Query.Sequence._packetToError (/snapshot-labs/snapshot-sequencer/
```

Reply:
If i change it to `votes.created` then i get `OperationalError: ER_WRONG_VALUE_COUNT_ON_ROW: Column count doesn't match value count at row 1`

Reply:
what will be the value of last_vote if there are no votes for a address

Reply:
> what will be the value of last_vote if there are no votes for a address

NULL

Reply:
Any idea how to solve the above? :(

Reply:
SQL error fixed